recipes/trio-util/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "trio-util" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 2f3a49085602885d2535fce43b17602c55a8fe98045a33c92e9c511054b20352
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - async_generator
+    - trio >=0.11.0
+
+test:
+  imports:
+    - trio_util
```

can we try to add ``pip check`` here please?
```suggestion
    - trio_util
  requires:
    - pip
  command:
    - pip check
```

---

Sure - I think that's a good idea. Any reason it's not part of the example template?